cookbooks/fb_init_sample/recipes/default.rb

```diff
@@ -128,12 +128,18 @@
 end
 
 unless node.centos6?
-  include_recipe 'fb_apcupsd'
+  # not packaged in C10 and above
```

Comment:
This is tracked in https://bugzilla.redhat.com/show_bug.cgi?id=2337139 for the record